README.md

```diff
@@ -55,6 +59,29 @@ However, you may want to install the latest version from the [GitHub repository]
 To run NPBench with DaCe, you have to select as framework (see details below)
 either `dace_cpu` or `dace_gpu`.
 
+
+### DPNP
+
+With `dpnp` it is strongly recommended to use `conda` instead of `pip` for its dependency on intel packages. 
+Refer to this 
+[LINK](https://intelpython.github.io/dpnp/quick_start_guide.html#building-for-custom-sycl-targets) to know more 
+about building custom SYCL targets or installing `dpnp` package from intel channel.
+
+Unlike the pip installation, with conda it is advisable to try installing all packages at once.
+Edit the `environment.yml` to include packages and optional dependencies (e.g. hardware-dependent frameworks
+or utilities such as `ipython`). Then type:
+
+``` bash
+$ conda env create -f environment.yml    # environment.yml contains all the right dependencies
+$ python -m pip install pygount          # Only dependency not distributed with conda
+```
+If you are behind proxies, follow your system documentation. E.g. for pip you may have to specify:
+``` bash
+$ python -m pip --proxy=http://localhost:1234 install <...> # where "localhost:1234" is the value of the env var "HTTP_PROXY"
+```
```

Comment:
This is unnecessary to include in the instructions. It is a pip issue rather than an NPBench issue.